src/Operation/Find.php

```diff
@@ -349,6 +349,9 @@ private function createCommandDocument(): array
         // maxAwaitTimeMS is a Query level option so should not be considered here
         unset($options['maxAwaitTimeMS']);
 
+        // writeConcern is not relevant to evaluating the query plan
+        unset($options['writeConcern']);
```

Comment:
The Find operation has never documented a `writeConcern` option. Moreover, `createQueryOptions()` never creates a `writeConcern` key in its return array. This can be removed.